packages/client/src/pages/signIn/modules/form/FormSignIn.tsx

```diff
@@ -11,11 +12,21 @@ import { useSignInMutation } from '../../../../store/api/yadnex/auth/authApi'
 import { UserShortInfo } from '../../../../types/auth'
 import { onOauth } from '../../../../store/api/yadnex/auth/Oauth'
 
-
 const FormSignIn = () => {
   const [signIn] = useSignInMutation()
-  const handleSubmit = (data: UserShortInfo) => {
-    signIn(data)
+  const navigate = useNavigate()
+
+  const handleSubmit = async (data: UserShortInfo) => {
+    try {
+      const result: any = await signIn(data)
+      if ('data' in result) {
```

Comment:
А так как снизу не сработает? Выглядит более читаемо

```suggestion
      if (result.data) {
```